pkg/drivers/kic/oci/volumes.go

```diff
@@ -26,18 +28,49 @@ import (
 // DeleteAllVolumesByLabel deletes all volumes that have a specific label
 // if there is no volume to delete it will return nil
 // example: docker volume prune -f --filter label=name.minikube.sigs.k8s.io=minikube
-func DeleteAllVolumesByLabel(ociBin string, label string) error {
+func DeleteAllVolumesByLabel(ociBin string, label string) []error {
+	var deleteErrs []error
 	glog.Infof("trying to prune all %s volumes with label %s", ociBin, label)
 	if ociBin == Docker {
 		if err := PointToHostDockerDaemon(); err != nil {
-			return errors.Wrap(err, "point host docker-daemon")
+			return []error{errors.Wrap(err, "point host docker-daemon")}
 		}
 	}
-	cmd := exec.Command(ociBin, "volume", "prune", "-f", "--filter", label)
+
+	vs, err := allVolumesByLabel(ociBin, label)
+	if err != nil {
+		glog.Infof("error listing volumes by label %q: %v", label, err)
```

Comment:
return?

Reply:
then reason I am not returning is because I wanna do a prune in the end anyways.
prune would not work on the in-use volumes. so at first I try delete them without prune
and use prune as last effort.

Reply:
based on your review, I ended up breaking this function to two functions. so I can return on errors better